src/client/components/SampleComponent/SlideshowImageComponent/SlideshowImageComponent.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './SlideshowImageComponent.styles.css';
+import ArrayOfImages from '../../../assets/data/ArrayOfImages';
+
+export default function SlideshowImageComponent() {
+    return (
+      <div ClassName="image_component">
```

Comment:
Avoid generic names like image_component.

Reply:
I will make sure to change and not repeat this style again.